Changes.md

```diff
@@ -1,6 +1,12 @@
 0.58.x.x
 ========
 
+Improvements
+------------
+
+- Catalogue : Added <kbd>Ctrl-D</kbd> shortcut to duplicate selected images (#3545).
+- Viewer : Added <kbd>Ctrl-D</kbd> shortcut to duplicate currently viewed image when viewing the output of Catalogue node (#3545).
```

Comment:
We should update the shortcuts section of the documentation to include this one too.

Reply:
Thanks for remembering! Updated [here](https://github.com/GafferHQ/gaffer/pull/3871/commits/e9a7948619376ae3017e59900a5460cd24e14dee).